docs/openapi/api-statistics-interop-v1.yaml

```
@@ -0,0 +1,173 @@
+openapi: 3.0.3
+info:
+  title: 'E-Service Monitor Interop Statistics service'
+  version: 1.0.0
+  x-summary: 'E-Service Monitor Statistics service'
+  description: OpenAPI specification for E-Service Monitor's operation services
+  termsOfService: https://www.e-serviceurl-example.com/terms
+  contact:
+    name: contactToDefinePagoPa
+    email: toDefinePagoPa@pagopa.it
+  license:
+    name: Interop's licence
+    url: https://to-define/
+servers:
+  - url: https://to-define/
+    description: Development environment
+paths:
+  '/eservices/statistics/{eserviceRecordId}':
```

is it `statistics` path required considering this is the statistics service?

---

I wrote statistics in the path just because it is the only api that the service expose for now, and because it is what the the api returns. I can change it to something like telemetryData if you dont want it to be redundant.

---

considering that providing statistics is the purpose of the service, in my opinion this is enough
```suggestion
  '/eservices/{eserviceRecordId}':
```

We will properly name any future side endpoint

---

ok